src/main/java/frc/robot/Robot.java

```diff
@@ -11,129 +11,143 @@
 import org.littletonrobotics.junction.wpilog.WPILOGReader;
 import org.littletonrobotics.junction.wpilog.WPILOGWriter;
 
-import AKInput.AKInput;
-import edu.wpi.first.wpilibj.PowerDistribution;
-import edu.wpi.first.wpilibj.PowerDistribution.ModuleType;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
 
 /**
- * The VM is configured to automatically run this class, and to call the functions corresponding to
- * each mode, as described in the TimedRobot documentation. If you change the name of this class or
- * the package after creating this project, you must also update the build.gradle file in the
- * project.
- */
+* The VM is configured to automatically run this class, and to call the functions corresponding to
+* each mode, as described in the TimedRobot documentation. If you change the name of this class or
+* the package after creating this project, you must also update the build.gradle file in the
+* project.
+*/
 public class Robot extends LoggedRobot {
-  private final boolean SIMULATOR_INPUTS = true;
-  private final boolean REAL_TIMING = true;
-
-  private Command m_autonomousCommand;
-
-  private RobotContainer m_robotContainer;
-
-  /**
-   * This function is run when the robot is first started up and should be used for any
-   * initialization code.
-   */
-  @Override
-  public void robotInit() {
-    Logger.recordMetadata("StudentAllegro2024", "StudentAllegro2024"); // Set a metadata value
-
-    if (isReal() || SIMULATOR_INPUTS) {
-        Logger.addDataReceiver(new WPILOGWriter()); // Log to a USB stick ("/U/logs")
-        Logger.addDataReceiver(new NT4Publisher()); // Publish data to NetworkTables
-        new PowerDistribution(1, ModuleType.kRev); // Enables power distribution logging
-    } else {
-        setUseTiming(REAL_TIMING); // Run as fast as possible
-        String logPath = LogFileUtil.findReplayLog(); // Pull the replay log from AdvantageScope (or prompt the user)
-        Logger.setReplaySource(new WPILOGReader(logPath)); // Read replay log
-        Logger.addDataReceiver(new WPILOGWriter(LogFileUtil.addPathSuffix(logPath, "_sim"))); // Save outputs to a new log
+    
+    private Command m_autonomousCommand;
+    
+    private RobotContainer m_robotContainer;
+    
+    /**
+    * This function is run when the robot is first started up and should be used for any
+    * initialization code.
+    */
+    @Override
+    public void robotInit() {
+        Logger.recordMetadata("Project", "StudentAllegro2024"); // Set a metadata value
+        
+        switch (Constants.ROBOT_MODE) {
+            case REAL:
+            Logger.addDataReceiver(new WPILOGWriter());
+            Logger.addDataReceiver(new NT4Publisher());
+            
+            // new PowerDistribution(1, ModuleType.kRev); // Enables power distribution logging
```

Comment:
Why did you disable PGP logging?

Reply:
It seemed as if it was already logging power distribution without that line. I can add it back if need be. It just seemed very odd to me to be making an instance of a class on one line like that.